src/Components/Progress.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@testing-library/react";
+import Progress from "./Progress";
+
+describe("Progress", () => {
+  it("renders total", () => {
+    render(<Progress goal={246} total={123} />);
+
+    expect(screen.getByRole("heading", { level: 2 }).textContent).toEqual(`Raised $123 of $246`);
```

Comment:
If we are querying by role, the instructions should probably specify to add the role attribute (or does the bootstrap code automatically do that?)

Reply:
^ Sorry for this comment. I looked up `getByRole` and I now actually (sort of) understand what it does and why it is not necessary to set the role attributes. You can ignore.